src/lax.js

```diff
@@ -581,8 +581,8 @@
         })
       }
 
-      addElements = (selector, transforms, options) => {
-        const domElements = document.querySelectorAll(selector)
+      addElements = (selector, transforms, options, elements) => {
```

Comment:
To reduce the API, I'd suggest changing the approach to accept a selector or an array of elements in the first parameter, e.g.
```suggestion
      addElements = (selectorOrElements, transforms, options) => {
```

Then we can check if `typeof selectorOr` is `string`. If so, call `document.querySelectorAll`. Otherwise, assign it to `domElements`.

Reply:
I wasn't sure about overriding `selector`, as it's passed down to `new LaxElement` below.  I only see it used for the `removeElements` method. Looks like we would still need a unique key passed in. Thoughts?

Reply:
How about:
```
      addElements = (selector, transforms, options) => {
        const domElements = options.domElements || document.querySelectorAll(selector) 

        domElements.forEach((domElement, i) => {
          this.elements.push(new LaxElement(selector, this, domElement, transforms, i, options))
        })
      }
```

Then we don't increase the number of arguments.

You'd still need a unique selector to be able to remove the elements. This should be specified in the docs.